app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -8,11 +8,14 @@ define(function (require, exports, module) {
   'use strict';
 
   var _ = require('underscore');
+  var p = require('lib/promise');
```

Comment:
Can you move this line down between `var Notifier...` and `var ProfileErrors...`?